plugins/inputs/t128_peer_path/t128_peer_path.go

```diff
@@ -74,6 +76,17 @@ func (*T128PeerPath) Description() string {
 // Init sets up the input to be ready for action
 func (plugin *T128PeerPath) Init() error {
 
+	var query string
+	if plugin.ExcludeHostname {
+		if _, found := peerPathTags["adjacentHostname"]; found {
+			delete(peerPathTags, "adjacentHostname")
+		}
+	} else {
+		if _, found := peerPathTags["adjacentHostname"]; !found {
+			peerPathTags["adjacentHostname"] = "adjacentHostname"
```

Comment:
This manipulation of global tags is no good. If someone uses two peer path inputs in the same telegraf instance, they can conflict. The input needs to operate on its own set of tags.